ykllvmwrap/src/jitmodbuilder.cc

```diff
@@ -1312,6 +1303,9 @@ class JITModBuilder {
                 (IID == Intrinsic::lifetime_end))
               continue;
 
+            if (IID == Intrinsic::frameaddress)
```

Comment:
Why do we skip this one?

Reply:
This isn't needed in the compiled trace as it's only passed into the control point which goes away as well in a compiled trace.

Reply:
Can we add a quick comment for our future selves?

Reply:
Fixed in [f689da1](https://github.com/ykjit/yk/pull/578/commits/f689da1d385791e1c7a821174b1712ddb0dd18b9)